pandas/tests/series/test_alter_axes.py

```diff
@@ -214,17 +214,6 @@ def test_reorder_levels(self):
         expected = Series(np.arange(6), index=e_idx)
         assert_series_equal(result, expected)
 
-        result = s.reorder_levels([0, 0, 0])
```

Comment:
can you add a test that raises

Reply:
It is the new ``test_set_index_duplicate_names`` (the constructor itself raises).